src/WebUI/Components/KubeSummary.tsx

```diff
@@ -21,16 +20,27 @@ import "azure-devops-ui/Label.scss";
 import { DaemonSetListingComponent } from "./DaemonSetListingComponent";
 import { StatefulSetListingComponent } from "./StatefulSetListingComponent";
 import { PodsComponent } from "./PodsComponent";
+import { Filter, IFilterState, FILTER_CHANGE_EVENT, IFilterItemState } from "azure-devops-ui/Utilities/Filter";
+import { KubeResourceType } from "../../Contracts/KubeServiceBase";
+import { FilterComponent } from "./FilterComponent";
+import { Tab, TabBar, TabContent } from "azure-devops-ui/Tabs";
+import { ObservableValue } from "azure-devops-ui/Core/Observable";
+import { HeaderCommandBarWithFilter } from 'azure-devops-ui/HeaderCommandBar';
 
 const workloadsPivotItemKey: string = "workloads";
 const servicesPivotItemKey: string = "services";
+const filterToggled = new ObservableValue<boolean>(false);

```

Comment:
we should see to show this filter icon or not if there is no data in workloads or services, this is inline with your other PR zero data